### DISCUSSION TITLE
Calendarios de actividades

### DISCUSSION THREAD

**Discussion by JoseEduardoVelazquezCN** [Launch-X-Latam/OnBoarding#14] [Q&A] [answered]
Hola, no encuentro el calendario de actividades, recuerdo verlo pero no dónde 🐧
Alguien sabe dónde se encuentra¿?

**Comment by devonent** [comment, marked as answer]
## Supongo que te refieres al siguiente calendario:
![image](https://user-images.githubusercontent.com/87509978/192867804-e56decef-15a2-40be-a1f3-28d99115fb2d.png)

**Reply by JoseEduardoVelazquezCN** [reply]
Justo ese muchas gracias ✌️
